stringscore_test.go

```diff
@@ -43,7 +43,7 @@ func TestScore(t *testing.T) {
 
 func TestExactMatchIsPrefferedOverFuzzyMatch(t *testing.T) {
 	query := "backend"
-	fuzzyScore := stringscore.Score("vendor/github.com/coreos/go-oidc/key/manager.go", query)
+	fuzzyScore := stringscore.Score("vendor/github.com/gorilla/websocket/conn_read_legacy.go", query)
```

Comment:
why this change?

Reply:
backend doesn't match it (ie score is 0). So wasn't actually testing what it said it was. Instead I took the longest fuzzy match I could find in our backend codebase.